### DISCUSSION TITLE
Plants should not age if placed on water

### DISCUSSION THREAD

**Discussion by QuantumKittens** [OpenRCT2/OpenRCT2#15738] [Feature requests]
**OS:** macOS
**Version:** 0.3.3
**Commit/Build:** 3f65f282d on HEAD

It seems that gardens will wilt after some time even if they are placed directly on water. This is unrealistic as they do not appear to have any kind of soil beneath them, as can be seen by the fact that placing a handyman on them causes him to drown.

Plant aging should therefore not occur for plants that are placed directly on water.

![bug](https://user-images.githubusercontent.com/45405783/125661662-be62313f-8845-4a3c-b350-fcf788048b07.png)

- [ ] Reproducible in RCT2 (vanilla)? unknown
- [ ] Specific to multiplayer? no

**Comment by IntelOrca** [comment]
If we disable ageing for flowers on water, it should probably only apply to the flowers that aren't within a base block.

**Comment by ZehMatt** [comment]
It would also a bit unrealistic to have plants sitting on water, a lot of plants would actually die if you don't supply sufficient oxygen for the roots to breath. Also plants normally don't float on top, only very few exceptions actually apply here.

**Reply by Derpercoaster** [reply]
I mean, in this game, you can build coasters that break the laws of physics, build towers made purely of water, violate every ethical law ever written to steal every last dollar from every guests' bank account, and handymen carry lawnmowers in their pockets. Plants floating on the surface of water, and not dying, sounds reasonable to me.
